prometheus_client/multiprocess.py

```diff
@@ -92,10 +92,11 @@ def _accumulate_metrics(metrics, accumulate):
             sample_timestamps = defaultdict(float)
             buckets = defaultdict(lambda: defaultdict(float))
             samples_setdefault = samples.setdefault
+            generate_pidless_key = lambda x, y: (x, tuple(l for l in y if l[0] != 'pid'))
```

Comment:
Rather than `x, y` can we keep `name, labels` so we know what each variable is? It would also be fine to move this to a real func instead of a lambda if there are more name conflicts.

Reply:
Just a quick bump on this request, otherwise good to merge.